docs/apis-tools/tasklist-api-rest/tasklist-api-rest-authentication.md

```diff
@@ -5,63 +5,137 @@ sidebar_position: 2
+   ```shell
+   curl --location --request POST 'http://localhost:18080/auth/realms/camunda-platform/protocol/openid-connect/token' \
+   --header 'Content-Type: application/x-www-form-urlencoded' \
+   --data-urlencode "client_id=${CLIENT_ID}" \
+   --data-urlencode "client_secret=${CLIENT_SECRET}" \
+   --data-urlencode 'grant_type=client_credentials'
+   ```
+5. A successful authentication response looks like the following:
+   ```json
+   {
+     "access_token": "<TOKEN>",
+     "expires_in": 300,
+     "refresh_expires_in": 0,
+     "token_type": "Bearer",
+     "not-before-policy": 0
+   }
+   ```
+6. Capture the value of the `access_token` property and store it as your token.
+
+See the [Tasklist Configuration - Authentication](/self-managed/tasklist-deployment/tasklist-authentication.md#identity) documentation for more information about this authentication method.
+
+</TabItem>
+
+</Tabs>
+
+## Using a token
+
+Send the captured token as an authorization header in each request: `Authorization: Bearer <TOKEN>`.
+
+For example, to call the Tasklist API's ["Search tasks" endpoint](./specifications/search-tasks.api.mdx), make the following request against the target Tasklist environment:
+
+<Tabs groupId="using-a-token" defaultValue="saas" queryString values={
+[
+{label: 'SaaS', value: 'saas' },
+{label: 'Self-Managed', value: 'self-managed' },
+]}>
+
+<TabItem value='saas'>
+
+:::tip
+The URL of the Tasklist API, represented below by the `${CAMUNDA_TASKLIST_BASE_URL}` variable, can be captured when creating an API client. It can also be constructed as `https://${REGION}.tasklist.camunda.io/${CLUSTER_ID}`.
+:::
+
+</TabItem>
+
+<TabItem value='self-managed'>
+
+:::tip
+The URL of the Tasklist API, represented below by the `${CAMUNDA_TASKLIST_BASE_URL}` variable, is configured in your Self-Managed installation. The default value is `http://localhost:8082`.
+:::
+
+</TabItem>
+
+</Tabs>
 
 ```shell
-curl -X POST --header 'content-type: application/json' --data '{"client_id": "<client-id>", "client_secret":"<client-secret>","audience":"<audience>","grant_type":"client_credentials"}' https://<authorization server url>
+curl --request POST ${CAMUNDA_TASKLIST_BASE_URL}/v1/tasks/search \
+   --header "Authorization: Bearer ${TOKEN}" \
+   --header 'Content-Type: application/json' \
+   --data-raw '{}'
 ```
 
-If the authentication is successful, the authorization server sends back the access token, when it expires, scope, and type:
+A successful response would include [matching tasks](./specifications/search-tasks.api.mdx). For example:
 
 ```json
-{
-  "access_token": "ey...",
-  "scope": "...",
-  "expires_in": 86400,
-  "token_type": "Bearer"
-}
+[
+  {
+    "id": "12345",
+    "name": "Do something",
+    ...
+  }
+]
 ```
 
-## Authentication for Self-Managed cluster
+## Token expiration
```

Comment:
Similar to my other comment - do we need to mention the change in default value? 86400 vs. 300?

Reply:
I don't think so. That strays into environment configuration more than "interacting via API," IMO. If you disagree, I'm open to suggested content here.

Reply:
Can you raise this with the Tasklist team to see if this default needs to be communicated somewhere?

Reply:
it's an identity thing. Mentioned at https://camunda.slack.com/archives/CRQAL5A1M/p1724356189900659.

Reply:
> It’s configurable in Keycloak at client level, or at realm level. You need to access Keycloak as an admin user, and then you can configure it.